docker-compose.yml

```diff
@@ -7,18 +8,33 @@ redis:
   image: redis
 elasticsearch:
   image: elasticsearch
-web:
-  build: .
+
+# Web server, routing /api and / to server, client respectively
+nginx:
+  build: ./scripts
+  dockerfile: Dockerfile.nginx
+  links: [client, server]
+  ports: [80]
+
+# Rails server
+server: &rails
+  links: [postgres, redis, elasticsearch]
+  build: ./server
   volumes:
-    - ./:/opt/hummingbird/
-  ports:
-    - 3000:3000
-    - 4200
-    - 49152:49152
-  links:
-    - postgres
-    - redis
-    - elasticsearch
+    - ./server:/opt/hummingbird/server
   environment:
     RAILS_ENV: development
 
+# Sidekiq
+sidekiq:
+  <<: *rails
+  command: [sidekiq]
+
+# Ember server
+client:
+  build: ./client
+  volumes:
+    - ./client:/opt/hummingbird/client
+    - /opt/hummingbird/client/node_modules
```

Comment:
Because we mount over `/opt/hummingbird/client` we end up with a complete lack of `node_modules` directory in that and so I tried putting it on a data volume to keep it around.  I think it might also have a side benefit of caching the partial results of failed installs to speed up the second (and third and fourth and fifth) tries.  NPM confirmed for shit.